hushline/__init__.py

```diff
@@ -35,12 +32,14 @@ def handle_rate_limit_exceeded(e):
     app.config["SESSION_COOKIE_SAMESITE"] = "Lax"
     app.config["PERMANENT_SESSION_LIFETIME"] = timedelta(minutes=30)
 
-    ssl_cert = "/etc/mariadb/ssl/fullchain.pem"
-    ssl_key = "/etc/mariadb/ssl/privkey.pem"
+    # Conditional SSL configuration based on environment
+    if os.getenv("FLASK_ENV") == "production":
```

Comment:
Maria DB code sneaking back in

Reply:
Yeah, I'm going to address that in a separate PR.